src/index.js

```diff
@@ -200,26 +206,26 @@ async function templateDir(from, to, useTS) {
 
 /**
  * @param {string} to
- * @param {'yarn' | 'npm'} packageManager
+ * @param {'yarn' | 'npm' | 'pnpm'} packageManager
```

Comment:
Looks like `@antfu/install-pkg` also supports bun, not that I'm really picky on types, but it'd help to know it's supported.

Reply:
fixed, https://github.com/preactjs/create-preact/pull/16/commits/06b261820e69e9b18f659a8d5f526f56bf3b0e0c#diff-bfe9874d239014961b1ae4e89875a6155667db834a410aaaa2ebe3cf89820556R217